pkg/translator/loki/convert.go

```diff
@@ -118,3 +133,33 @@ func timestampFromLogRecord(lr plog.LogRecord) time.Time {
 
 	return time.Unix(0, int64(pcommon.NewTimestampFromTime(timeNow())))
 }
+
+func severityToLevel() map[plog.SeverityNumber]string {
```

Comment:
Can we make that a var?

Reply:
Yes of course, but if I follow your second comment, I can just remove that.

Reply:
But I guess you still need this `SeverityNumber` -> `string` mapping, so the attribute will look like, e.g. `level: ERROR`, right?

Reply:
Indeed, I mixed up between severity specification and level attribute value. You're right.